vhd-format-lwt.opam

```diff
@@ -7,14 +7,15 @@ doc: "https://mirage.github.io/ocaml-vhd/"
 bug-reports: "https://github.com/mirage/ocaml-vhd/issues"
 depends: [
   "ocaml" {>= "4.02.3"}
-  "cstruct"
+  "cstruct" {< "7.0.0"}
```

Comment:
Is this upper bound really needed?

Reply:
I don't really know the release process of `xapi`. I don't really agree to add this constraint when we care a lot about `cstruct` (and its release process) but it seems fine. May be it does not really help the solver at least.

Reply:
The bound is not needed at all for the xapi project. I don't remember the exact reason why I added it, I don't have a strong opinion about it.

Reply:
I will delete this constraint on the release PR, I think it does not really help the solver.